java/lang/security/audit/xxe/documentbuilderfactory-disallow-doctype-decl-missing.yaml

```diff
@@ -94,62 +97,64 @@ rules:
       - by-side-effect: true
         pattern-either:
           - patterns:
-            - pattern-either:
```

Comment:
I think you inadvertently pushed changes to this rule which is unrelated to your PR. Could you please revert?

Reply:
My bad. I reverted the changes.